app/helpers/dashboard_helper.rb

```diff
@@ -15,6 +15,8 @@ def dashboard_actions
 
   def render_widget(widget)
     render(:partial => widget.template, :locals => widget.data)
+  rescue ActionView::MissingTemplate
+    _("Missing template '%{template}' for widget '%{widget}'. The widget or a plugin providing it was probably removed") % { :widget => widget.name, :template => widget.template }
```

Comment:
I've removed the sentence itself, as in theory there might be more issues that caused problem, and we can leave the explanation to the errors wiki